src/components/NavBar/index.jsx

```diff
@@ -1,8 +1,40 @@
-import React from 'react'
+import React, { useState, useEffect } from 'react'
+import { Link } from "react-router-dom";
 
 const NavBar = () => {
+  const [darkMode, setDarkMode] = useState(false);
+
+  useEffect(() => {
+    const root = window.document.documentElement;
+    if (darkMode)
+    {
+      root.classList.add('dark');
+    }
+    else
+    {
+      root.classList.remove('dark');
+    }
+  }, [darkMode]);
+
+  const toggleDarkMode = () => {
+    setDarkMode(!darkMode);
+  };
+
+
   return (
-    <div>NavBar</div>
+    <div className='sticky'>
```

Comment:
use flex here to center the child div inside it and keep it's width as w-screen

Reply:
Done